__tests__/app.test.js

```diff
@@ -33,4 +33,4 @@ describe('GET/api/topics', () => {
             
     })
 })
-})
+});
```

Comment:
the `.forEach` on [line 29](https://github.com/leahbanks/nc_news_project/blob/cecdf64fa46a59ef99f834e3184a85902a84a6bc/__tests__/app.test.js#L29) needs to be protected with a length check on the array. If the array has no length and is not populated, this test will currently give you a false positive.